### DISCUSSION TITLE
PocketHost outage Jan 18, 2023

### DISCUSSION THREAD

**Discussion by pyloverm** [pockethost/pockethost#165] [Q&A] [answered]
my application pockethost loads in a loop, I can't access it, neither by web nor by clien js.
im getting 'ERR_TIMED_OUT'.

**Comment by benallfree** [comment]
@pyloverm What is the subdomain?

**Reply by pyloverm** [reply]
https://poor-camera.pockethost.io

**Reply by benallfree** [reply]
[https://poor-camera.pockethost.io/_](https://poor-camera.pockethost.io/_) is responding now.

**Comment by larsniet** [comment]
Same here https://forwardfuture.pockethost.io

**Reply by benallfree** [reply]
[https://forwardfuture.pockethost.io/_](https://forwardfuture.pockethost.io/_) is responding now.

**Comment by benallfree** [comment]
There is an outage. Looking into it.

**Comment by benallfree** [comment]
@larsniet @pyloverm Something strange is happening with your instances. I cannot get your instances to load, but the instances from my account launch fine. Looking into it.

**Reply by pyloverm** [reply]
I've been doing a lot of testing lately, too many requests from a single ip could it be the cause ?

**Reply by benallfree** [reply]
@pyloverm Unlikely. Still investigating.

**Comment by benallfree** [comment, marked as answer]
We are back up. Something caused a major CPU spike, still investigating the cause.

<img width="978" alt="image" src="https://user-images.githubusercontent.com/1068356/213183035-88bfbd7d-6126-41d3-83a3-2b75e0050548.png">
